Dockerfile

```diff
@@ -18,13 +18,9 @@ RUN apt-get update && apt-get install -y python3.7
 RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2
 RUN apt-get install -y python3-pip
 
-# aws
-RUN pip3 install awscli --upgrade
-RUN pip3 install --upgrade 'git+git://github.com/chanzuckerberg/s3mi.git'
-
 # bioinformatics
 RUN apt-get install -y samtools bowtie2 vsearch
-RUN pip3 install pysam biopython
```

Comment:
pysam no longer build successfully

fortunately it is not yet needed

we hope in future it gets fixed